config.lua

```diff
@@ -110,3 +110,21 @@ QBConfig.Server.CheckDuplicateLicense = true -- Check for duplicate rockstar lic
 QBConfig.Server.Permissions = { 'god', 'admin', 'mod' } -- Add as many groups as you want here after creating them in your server.cfg
 
 QBConfig.NotifyPosition = 'top-right' -- 'top' | 'top-right' | 'top-left' | 'bottom' | 'bottom-right' | 'bottom-left'
+
+---@alias TableName string
+---@alias ColumnName string
+---@type table<TableName, ColumnName>
+CharacterDataTables = {
```

Comment:
Config = {}
Config.CharacterDataTables = ...

I like having the Config object to signify that this isn't supposed to be a mutable global variable, but rather an immutable field set in config